### PR TITLE
Enables bitcode support for TLSphinx

### LINKED ISSUE #36: Enable bitcode support
I wonder if you're building sphinxbase/pocketsphinx on your own or with the `build_iphone.sh` script provided by pocketsphinx-ios-demo? I just PR'd them with the _smallest change ever_ just to enable bitcode (eg. -embed-bitcode). 
Would you consider updating your scripts and thus enabling bitcode support form TLSphinx as well?

### PR DESCRIPTION
**Pull request by coredumped**
Should fix #36 
Provides sphinx binaries with bitcode support and also allows bitcode compilation for TLSphinx.
Includes a few utility scripts to rebuild sphinxbase and pocketsphinx when new versions are released by CMU's team.
Binaries provided are still pre5alpha, but including new releases should be a lot easier in the future.

### CODE REVIEW

File: Sphinx/builder/README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+```
+cd Sphinx/builder/
+./install.sh
+```
+
+
+---
+## Manual Compilation
+If possible please try automatic compilation when possible, manual compilation is not for the feint of heart yet it might help in scenarios where you want to build CMU Sphinx Base and Pocket Sphinx from their respective master branches.
```
Comment (BrunoBerisso) [inline Sphinx/builder/README.md]: There is a typo: _feint_ should be _faint_

**Comment by BrunoBerisso** [comment]
Thank you very much for this PR :) Really apreciaste your work.

I found something strange when checking the binaries here. There seems not to have the bit code included in the libraries. I'm running this tests (I test all the fat binaries, here only mention `libsphinxbase.a` for short):

```
$ file libsphinxbase.a
libsphinxbase.a: Mach-O universal binary with 5 architectures
libsphinxbase.a (for architecture armv7):	current ar archive random library
libsphinxbase.a (for architecture armv7s):	current ar archive random library
libsphinxbase.a (for architecture i386):	current ar archive random library
libsphinxbase.a (for architecture x86_64):	current ar archive random library
libsphinxbase.a (for architecture arm64):	current ar archive random library
$ otool -l libsphinxbase.s | grep bitcode
$ otool -l libsphinxbase.s | grep __LLVM
```

The calls to `otool` doesn't give any output so I try with [this tool](https://github.com/JDevlieghere/LibEBC) with more clear output:
```
$ ./ebcutil libsphinxbase.a
Error: No bitcode section in bio.o
```

Also the date of the fat library for pocketsphinx doesn't match the date for all the architectures [here](https://github.com/coredumped/TLSphinx/tree/development/Sphinx/lib/pocketsphinx), maybe you forgot to rebuild the fat lib?

Thanks again!

**Comment by coredumped** [comment]
Let me check what happened with the Fat one....

**Comment by coredumped** [comment]
Ok, this is what I found... otool does not work recursively, that is... it will look for symbols in the fat binary itself but not in any embedded static libs thus it won't show bitcode information.
If you run:
```
for dir in * 
do 
 if [ -d $dir ]; then
  echo
  echo "${dir}:"
  otool -l ${dir}/libsphinxad.a | grep bitcode | head -1
  otool -l ${dir}/libsphinxad.a | grep __LLVM | head -1
  otool -l ${dir}/libsphinxbase.a | egrep bitcode | head -1
  otool -l ${dir}/libsphinxbase.a | egrep __LLVM | head -1
 fi
done
```
You'll get this output
```
arm64:
  sectname __bitcode
   segname __LLVM
  sectname __bitcode
   segname __LLVM

armv7:
  sectname __bitcode
   segname __LLVM
  sectname __bitcode
   segname __LLVM

armv7s:
  sectname __bitcode
   segname __LLVM
  sectname __bitcode
   segname __LLVM

i386:

x86_64:
```
The reason i386 and x86_64 does not show bitcode information is because it is till not supported for macOS/App Store apps.

My guess is that ebcutil shows no bitcode information on bio.o because it is testing the one in either i386 and x86_64 and is correctly stating that those do not have bitcode support.

The case for pocketsphinx's fat binary was that my script did not copied it due to a bug (actually a missing cp command), I'll be committing a fix shortly and also a new set binaries.

**Comment by coredumped** [comment]
According to [ebcutil](https://github.com/JDevlieghere/LibEBC)...
> You can also use ebcutil to check whether a given binary or library contains bitcode. It prints the first object encountered without bitcode and exits with a non-zero exit status.

Which matches with my previous comment.

**Comment by BrunoBerisso** [comment]
Awesome! Thanks

**Comment by coredumped** [comment]
My pleasure

**Pull request merged**